DataFormats/PatCandidates/interface/Jet.h

```diff
@@ -71,6 +71,7 @@ namespace pat {
   typedef std::vector<edm::FwdPtr<reco::BaseTagInfo> > TagInfoFwdPtrCollection;
   typedef std::vector<edm::FwdPtr<reco::PFCandidate> > PFCandidateFwdPtrCollection;
   typedef std::vector<edm::FwdPtr<CaloTower> > CaloTowerFwdPtrCollection;
+  typedef std::vector<edm::Ptr<pat::Jet> > JetPtrCollection;
```

Comment:
I'm going back and forth on this. If we want CandPtr then we have to continually dynamic cast to get the pat::Jets, and this may be confusing to the users.